### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by SevKrok**

### CODE REVIEW

**Review by Oxbay — APPROVED** [review]
Great work)

File: app/main.py [hunk 1]
```diff
@@ -2,4 +2,73 @@ class Cargo:
...
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None
+    ) -> None:
+        if coords is None:
+            coords = [0] * 2
```
Comment (fsocie7y) [inline app/main.py:14]: you can write this in one line.  Use a single composite `if` statement

File: app/main.py [hunk 2]
```diff
@@ -2,4 +2,73 @@ class Cargo:
...
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None
+    ) -> None:
+        if coords is None:
```
Comment (fsocie7y) [inline app/main.py:42]: the same here, and also u can move it into super().__init__()

**Review by fsocie7y — CHANGES_REQUESTED** [review]

**Review by fsocie7y — APPROVED** [review]
well done ^_^

**Pull request closed without merging**